### DISCUSSION TITLE
How to use the project module?

### DISCUSSION THREAD

**Discussion by ssardina** [idrave/FOND-ASP#14] [Q&A] [answered]
I noticed you did a project that you can install via `setuptools` which seems great (I have never used that in Python though!)

can we explain a bit how to use it? How were you using that Ivan?

**Comment by idrave** [comment]
Yes! the setup.py is used to setup the python module. This is done when running:
`pip install -e .`
as in the setup explained in the README. Once this is done, the project should be available as a module in the python environment.

**Comment by ssardina** [comment]
Interesting. What does it mean to be available as a module? I can just import it from any Python file I guess?

I ask because the README says it will be installed in `src/fcfondplanner`, so it will not go system wide or user wide right?

**Comment by ssardina** [comment, marked as answer]
Ohh I guess it will be added to the Python system environment but it will live in that folder `src/fcfondplanner`?

Sorry for my ignorance.. I never went so far with Python... But there is always a first time eh..

**Reply by idrave** [reply]
Sorry! I didn't see this. Yes exactly, it will be accessible from anywhere while you are within the python environment where you setup the module. Actually, I think the path is added to the `sys.path` variable within the environment

**Reply by ssardina** [reply]
You are totally right, now I get it. So it is indeed added to `sys.path`:

![image](https://user-images.githubusercontent.com/2368384/132267980-94db6800-ff47-4775-81e8-2511b138673a.png)

That is why when I modified the code it is already visible, because Python has registered the folder as one of its path. Got it now, thanks!
